src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentService.kt

```diff
@@ -43,13 +49,28 @@ class IncidentService(
       incidentDateTime = LocalDateTime.of(this.incidentDate, this.incidentTime),
       reportedStaff = this.reportingStaff.toStaff(),
       reportedDateTime = LocalDateTime.of(this.reportedDate, this.reportedTime),
-    )
-
-  fun uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Staff.toStaff() =
-    Staff(
-      staffId = id,
-      firstName = firstName,
-      lastName = lastName,
-      username = accounts.maxByOrNull { it.type }?.username ?: "unknown",
+      staffParties = this.staffParties().map { it.staffParty().toStaff() },
+      offenderParties = this.offenderParties().map { it.offenderParty().toOffender() },
```

Comment:
These lines look a bit faffy to me - will see if I can improve them/rename in next PR to make it more obvious what's actually happening.